### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by cnu1812**
## Related Issue

- Fixes # (issue) #4 

## Proposed Changes 

- Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

## Additional Info

- Any additional information, context or references. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by cnu1812** [comment]
@shreemaan-abhishek

File: .github/ISSUE_TEMPLATES/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+- name: 🙋🤔‍Question/Doubt
+  url: https://www.youtube.com/c/DoKcommunity
+  about: Feel free to ask any questions on our Slack channel.
```
Comment (shreemaan-abhishek) [inline .github/ISSUE_TEMPLATES/config.yaml:5]: Why does this file has so less lines of code.

**Review by shreemaan-abhishek — CHANGES_REQUESTED** [review]

**Comment by shreemaan-abhishek** [comment]
Also, please add meaningful commit messages that explain what you did instead of commit messages like these:
image

**Comment by cnu1812** [comment]
This is how it looks like.
Screenshot 2022-03-19 182106
.

**Review by AbhijithGanesh — APPROVED** [review]

**Pull request merged**